test/SILOptimizer/target-const-prop.swift

```diff
@@ -5,7 +5,7 @@
 // RUN: %target-build-swift -O -module-name=test %s -o %t/a.out
 // RUN: %target-run %t/a.out | %FileCheck %s -check-prefix=CHECK-OUTPUT
 
-// REQUIRES: executable_test,swift_stdlib_no_asserts,optimized_stdlib
```

Comment:
are you sure about removing the `swift_stdlib_no_asserts` requirement?

Reply:
Well, the test passes for me locally (let's see what CI says) -- in case it works, would you agree it's desirable to remove that requirement? Or do you know why it's there in the first place?

Reply:
should be fine to remove it